legacy/tripal_pub/tripal_pub.install

```diff
@@ -54,11 +54,11 @@ function tripal_pub_install() {
   $obo_id = tripal_insert_obo('Tripal Publication', $obo_path);
   tripal_submit_obo_job(array('obo_id' => $obo_id));
 
-  tripal_pub_add_cvs();
-  tripal_pub_add_cvterms();
+  //tripal_pub_add_cvs();
+  //tripal_pub_add_cvterms();
 
   // add the custom tables
-  tripal_pub_add_custom_tables();
+  //tripal_pub_add_custom_tables();
```

Comment:
All of the commented out functions simply don't exist. I assume because we use the obo file to load cvs instead of manually adding them?